### DISCUSSION TITLE
2 Feature Requests, Please.

### DISCUSSION THREAD

**Discussion by dev-dev-15** [dailydotdev/daily#902] [Feature Requests]
**1. A section for news only:**.
Sometimes we just want to know what is going on in the tech industry, which means we need to read news articles, so I think it would be great if you could add a `News` button under the `Discover` section in the sidebar, which will of course take us to only news articles.
With AI it's very easy to tell if an article is news or not.

**2. The limit of 3 daily suggestions is not enough:**.
Well, at least for some users, I know you have your reasons why you set that limit, but I think it's good (for daily.dev and for users) to let some people post more than that.

Of course I don't know what kind of users you have, but I can guess two types, or make it three:
* The user who only reads articles and never posts.
* The user who reads articles and sometimes submits.
* The user who reads articles and also submits a lot of good quality articles and is obsessed with his reputation.

For the last type of user, I think you should let them submit more than 3 articles, which will make them happy and also mean more articles for the readers.

Of course you will set some guidelines for this, for example: the user must have at least 3000 reputation + must submit 3 articles a day for the last 20 days + no reports for his article - something like that.

Maybe give a badge to such users, `Publisher` or something like that.

**Comment by idoshamun** [comment]
Hi @0proboy0,

Thanks a lot for the suggestions. We'll take that into account :)
